website/content/api-docs/agent/check.mdx

```diff
@@ -238,6 +238,11 @@ The table below shows this endpoint's support for
   made to both addresses, and the first successful connection attempt will
   result in a successful check.
 
+- `UDP` `(string: "")` - Specifies a `UDP` to send datagrams against the value of `UDP`
+  (expected to be an IP or hostname plus port combination) every `Interval`. If
+  the datagram sending is successful or gets a timeout, the check is `passing`. 
+  Any other failing result is unsuccessful, the check is `critical`.
```

Comment:
```suggestion
- `UDP` `(string: "")` - Specifies a `UDP` IP address, hostname, port, or combination thereof. The check sends datagrams to the value specified at the interval specified in the `Interval` configuration. If the datagram is sent successfully or a timeout is returned, the check is set to the `passing` state.  The check is logged as `critical' if the datagram is sent unsuccessfully.
```
Just trying to be clearer about the actual value and how to understand the results.